packages/rsocket-ws-gateway/src/api/Gateway.ts

```diff
@@ -1,7 +1,11 @@
 import { MicroserviceApi } from '@scalecube/api';
 
+export type GatewayPluginType = (serviceCall: MicroserviceApi.ServiceCall, data: any, subscriber: any) => void;
```

Comment:
I would change this name `GatewayPluginType` to `RequestHandler`

I don't like `packages/rsocket-ws-gateway/src/createGatewayProxy.ts` there is room to improve

But it's good enough for now